lib/flipper/adapters/active_support_cache_store.rb

```diff
@@ -88,14 +88,14 @@ def get_all
       ## Public
       def enable(feature, gate, thing)
         result = @adapter.enable(feature, gate, thing)
-        @cache.delete(key_for(feature.key))
+        @cache.write(key_for(feature.key), @adapter.get(feature), @write_options)
```

Comment:
This does mean we have to go back to the underlying adapter immediately to read the new value, which might not be ideal.